docs/manual-of-auto-testing-codes.md

```diff
@@ -0,0 +1,40 @@
+# The manual of auto-testing codes
```

Comment:
@takeutak Any chance that you could use the default test script and then there's no need for this document?

Reply:
The doc was already put on this file <https://github.com/hyperledger/cactus/blob/main/packages/manual-of-auto-testing-codes.md>. I intended to move this file to docs directory, but I missed to delete the one on the original directory.
On another PR, I will delete the one on the original directory to fix it.

Reply:
@takeutak Thanks for letting me know. Do you have any plans to migrate to the default test scripts that get invoked by Lerna for the combined coverage reports?

Reply:
I do not have any plan yet. However, I think combining it with existing CI checking tools to make it easier to use is one of the ways to improve it.
I think it is appropriate to discuss this topic in another issue. Could you approve this PR for now?

Reply:
@takeutak Awesome, thanks.